CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+# Contributing to Patternfly React
+
+## Code of Conduct
+
+This project is governed by the [Contributor Covenant version 1.4][1].  All contributors and participants 
+agree to abide by its terms. To report violations, send an email to [patternfly@redhat.com][2].
+
+## Issue Contributions
+
+The [issue tracker](https://github.com/patternfly/patternfly-react/issues) is the preferred channel for bug reports, features requests, and submitting pull requests.
+
+### Issues and labels
+
+Our issue tracker utilizes several labels to help organize and identify issues. Here's what they represent and how we use them:
+
+- `bug` - A bug is a _demonstrable problem_ that is caused by the code in the repository. Please check if the issue has already been reported before reporting a new bug.
+- `duplicate` - A duplicate signifies that another issue encapsulates or restates this issue. Duplicates will be closed and referenced as such. 
+- `ehancement` - An ehancement is a feature request. Please provide as much detail as possible and consider whether your idea fits the scope and aims of this project.
+- `help wanted` - Issues we need or would love help from the community to resolve.
+- `pending response` - This label indicates that the issue is awaiting reply from a contributor or a maintainer.
+- `question` - Use the question label to indicate you are having a problem with something and would like feedback from the community.
+- `wontfix` - The issue is legitimate, but it is not something the team is currently able or willing to fix or implement. Issues with this label may be revisited in the future.
+
+## React Component Contribution Process
+
+React component contributions should observe the following high level process:
+
+1. Check the [Patternfly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) to see if your component meets the criteria for an addition to Patternfly.
+2. Create an `enhancement` issue to indicate you'd like to contribute this component to Patternfly React and notify the Patternfly React maintainer team.
```

Comment:
Updating my suggestion to include more details about the review process:

> Create an enhancement issue to indicate you'd like to contribute this component to PatternFly React. At this time, the PatternFly React design team will review the enhancement issue and verify that the component passes the decision tree. Only components that pass will be accepted into the repo. If the component does not pass, the team will work with you on identifying any sub-components that can be extracted that would be needed by more than one project.

Please review and let me know if this accurately captures what @maryclarke and @wrnkld and Rox will do. I don't want to make any promises. :-) 

Also let me know if this should be broken out somewhere else.